src/lib/forward.ts

```diff
@@ -119,33 +128,33 @@ export class EventForwarder {
     }
 
     {
-      const events: ZWaveNodeEvents[] = [
-        "value updated",
-        "value removed",
-        "interview failed",
-      ];
+      const events: ZWaveNodeEvents[] = ["value removed", "interview failed"];
       for (const event of events) {
-        node.on(event, (changedNode: ZWaveNode, args: any) =>
-          notifyNode(changedNode, event, { args })
-        );
+        node.on(event, (changedNode: ZWaveNode, args: any) => {
+          notifyNode(changedNode, event, { args });
+        });
       }
     }
 
     {
-      const events: ZWaveNodeEvents[] = ["value added", "value notification"];
+      const events: ZWaveNodeEvents[] = ["value updated", "value added"];
       for (const event of events) {
         node.on(event, (changedNode: ZWaveNode, args: any) => {
-          // include metadata and ccVersion in the response
-          const metadata = node.getValueMetadata(args);
-          args.metadata = metadata;
-          args.ccVersion =
-            node.getEndpoint(args.endpoint).getCCVersion(args.commandClass) ||
-            node.getEndpoint(0).getCCVersion(args.commandClass);
+          // only forward value events for ready nodes
+          if (!changedNode.ready) return;
           notifyNode(changedNode, event, { args });
         });
       }
     }
 
+    node.on(
+      "value notification",
```

Comment:
@marcelveldt  Why don't add this to the `ZwaveNodeEvents` ? I think that you should check for the node to be ready here too

Reply:
I did that because we dump that to a full state too but I guess this doesn't add any noticeable overhead. Adjusted it.